.travis.yml

```diff
@@ -51,3 +51,13 @@ script:
 
 after_success:
   - if [[ "$WITH_COVERAGE" == "true" ]]; then php composer/bin/test-reporter; fi
+
+deploy:
+  provider: releases
+  api_key: $GITHUB_API_KEY
```

Comment:
@maxjacobson 

It's probably best to create a token for a bot account (if you have one), right here: https://github.com/settings/tokens